### DISCUSSION TITLE
Build error for Windows targeting CY2024

### DISCUSSION THREAD

**Discussion by ronBublitz** [AcademySoftwareFoundation/OpenRV#411] [Build Issues] [answered]
Getting this error when building:
 "C:\OpenRV2.0\_build\main_executable.vcxproj" (default target) (1) ->
       "C:\OpenRV2.0\_build\src\bin\apps\rv\rv.vcxproj" (default target) (3) ->
       "C:\OpenRV2.0\_build\executables_with_plugins.vcxproj" (default target) (25) ->
       "C:\OpenRV2.0\_build\src\bin\imgtools\rvio\rvio.vcxproj" (default target) (120) ->
       "C:\OpenRV2.0\_build\src\lib\ip\OCIONodes\OCIONodes.vcxproj" (default target) (237) ->
       (ClCompile target) ->
         C:\OpenRV2.0\src\lib\ip\OCIONodes\OCIO1DLUT.cpp(24,15): error C2660: 'OpenColorIO_v2_3::Gp
       uShaderDesc::getTexture': function does not take 7 arguments [C:\OpenRV2.0\_build\src\lib\ip
       \OCIONodes\OCIONodes.vcxproj]

Using the repo as of March 13, 2024.
VS Studio 2022 Community

I was able to successfully build RV2.0 with the downloaded repo without any changes.  However, I wanted to test the changes when the CY2024 was specified (in /cmake/defaults/rv_options.cmake).  The error above is what was output after doing a clean build.

**Comment by bernie-laberge** [comment]
Hello @ronBublitz,
I am going to take care of fixing this VFX Reference Platform CY2024 because of this OCIOv2 API change.
Sorry about that.

**Comment by bernie-laberge** [comment, marked as answer]
FYI: The fix has been merged:
https://github.com/AcademySoftwareFoundation/OpenRV/pull/413

**Reply by ronBublitz** [reply]
Thank you!  It does build without errors now.
